cleanco/classify.py

```diff
@@ -27,7 +27,8 @@ def classification(source):
     for elf_code, values in Elf.items():
         source_match = getattr(Elf[elf_code][0], source)
         if source_match:
-            entity_codes = [Elf[elf_code][0].local_abbreviations][0].split(";")
+            entity_codes = entity_codes = [Elf[elf_code][0].local_abbreviations][0].split(";") + [Elf[elf_code][
```

Comment:
`[Elf[elf_code][0]` presumes there's only one language version, it seems to me, thus ignoring all the multi-language versions of codes. That misses a lot of abbreviations, e.g. for countries such as Belgium.